apps/web/components/Organization/Slice/EmailSignup/EmailSignup.tsx

```diff
@@ -96,6 +95,15 @@ const EmailSignup = ({ slice, marginLeft }: EmailSignupProps) => {
           'fieldIsRequired',
           'Þennan reit þarf að fylla út',
         )
+      } else if (
+        field.type === FormFieldType.NATIONAL_ID &&
+        field.required &&
+        !isValidNationalId(value)
```

Comment:
What if the field is not required? should it not validate that it is a national ID or empty? Something like this: (it's a national ID **or** it's an empty optional field)

    field.type === FormFieldType.NATIONAL_ID && !(isValidNationalId(value) || (!field.required && !value))